packages/sdk/contractkit/src/wrappers/Governance.ts

```diff
@@ -232,6 +232,14 @@ export class GovernanceWrapper extends BaseWrapperForGoverning<Governance> {
     }
   }
 
+  // function get support doesn't consider constitution parameteres that has an influence
+  // in the total of yes votes required
+  async getSupportWithConstution(proposalID: BigNumber.Value, constitution: BigNumber) {
```

Comment:
nit: call the param `constitutionThreshold` rather than consitution? 

I think it may make it a little easier to understand for people just learning